templates/default/apache_site.conf.erb

```diff
@@ -24,6 +24,14 @@
 
   ServerSignature Off
 
+  <IfModule mod_rewrite.c>
```

Comment:
I don't really agree on the use of IfModule, but since it was already in the original location I'll accept it for this PR.

If I have a uncommented directive, I want it to error if it's module isn't enabled, rather than silently fail